docs/css/homepage.css

```diff
@@ -0,0 +1,69 @@
+.homepage div[role="main"] {
+    width: 100%;
+}
+
+body.homepage div.container img {
+    border-width: 0;
+    margin: 0;
+    padding: 0;
+    background-color: transparent;
+}
+
+.homepage .top {
+    padding: 20px 0;
+    text-align: center;
+    color: #26282a;
+    width: 100%;
+    border-bottom: 1px solid #f1f1f5;
+}
+
+.homepage .top p {
+    font-size: 16px;
+    font-weight: 300;
+}
+
+.homepage .top img {
+    height: 62px;
+    width: 50px;
+    margin: 0;
+    padding: 0;
+}
+
+.homepage .top h1 {
+    color: #003abc;
```

Comment:
I keep thinking that these headings look like links.
Can you try this with `color: #26282a;`?